### PR TITLE
include=["TAGS"] for describe_task_definition

### PR DESCRIPTION
**Pull request by ipmb**
refs #3138 

First time working with `moto`. Is this the right approach for a fix? Let me know and I can apply it to `ecs.describe_service` and write tests.

### CODE REVIEW

File: moto/ecs/responses.py [hunk 1]
```diff
@@ -87,7 +87,10 @@ def list_task_definitions(self):
     def describe_task_definition(self):
         task_definition_str = self._get_param("taskDefinition")
         data = self.ecs_backend.describe_task_definition(task_definition_str)
-        return json.dumps({"taskDefinition": data.response_object, "failures": []})
+        resp = {"taskDefinition": data.response_object, "failures": []}
+        if "TAGS" in self._get_param("include", []):
+            resp["tags"] = self.ecs_backend.task_definitions[data.family][data.revision].tags
```
Comment (bblommers) [inline moto/ecs/responses.py:92]: Would be good to have a utility method for this, i.e. `self.ecs_backend.get_tags_for_task(..)`

**Review by bblommers — CHANGES_REQUESTED** [review]
Hi @ipmb, thanks for your PR! This looks like a good approach, yes - just a minor niggle to make it more readable.

Keep in mind that you need to run `black` (in Python3) for the build to pass: https://travis-ci.org/github/spulec/moto/builds/721908719

File: moto/ecs/responses.py (hunk 1 above)
Comment (ipmb) [reply moto/ecs/responses.py]: turns out there already is! I found `list_tags_for_resource`

**Review by bblommers — APPROVED** [review]
LGTM. Thanks @ipmb!

**Pull request merged**